### PR TITLE
Add billing summary to CLI

### PR DESCRIPTION
**Pull request by DifferentSC**
This PR includes
* Add fetching billing summary to PF CLI (`pf billing summary ...`)
* Add necessary tests

### CODE REVIEW

File: pfcli/billing.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+panel_formatter = PanelFormatter(
+    name="Billing",
+    fields = ['vm_name', 'price'],
+    headers = ['VM Name', 'Amount used in USD']
+)
+
+@app.command(help="summarize billing information")
+def summary(
+    organization_id: Optional[str] = typer.Argument(
```
Comment (ktaebum) [inline pfcli/billing.py:42]: `UUID`?

File: pfcli/billing.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        None,
+        '--organization',
+        '-o'
+    ),
+    project_id: Optional[str] = typer.Argument(
```
Comment (ktaebum) [inline pfcli/billing.py:47]: `UUID`?

File: test/service/client/test_billing.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import uuid
```
Comment (ktaebum) [inline test/service/client/test_billing.py:1]: copyright?

File: pfcli/billing.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
+from functools import reduce
```
Comment (ktaebum) [inline pfcli/billing.py:1]: copyright?

File: pfcli/billing.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    ),
+    month: int = typer.Argument(
+        ...
+    ),
+    day: Optional[int] = typer.Argument(
+        None
+    ),
+    organization_id: Optional[UUID] = typer.Option(
+        None,
+        '--organization',
```
Comment (ktaebum) [inline pfcli/billing.py:56]: `--organization-id`? could be confusing

File: pfcli/billing.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        '-o'
+    ),
+    project_id: Optional[UUID] = typer.Option(
+        None,
+        '--project',
```
Comment (ktaebum) [inline pfcli/billing.py:61]: `--project-id`? 
could be confusing

File: pfcli/billing.py [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    group_id = None
+    project_id = None
+    if not view_organization:
+        project_id = get_current_project_id()
+        if project_id is None:
+            secho_error_and_exit("'project_id' is not set!")
+    else:
+        group_id = get_current_group_id()
+        if group_id is None:
+            secho_error_and_exit("'group_id is not set!")
```
Comment (ktaebum) [inline pfcli/billing.py:73]: `Organization is not set`?

File: pfcli/billing.py [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+    )
+):
+    "Summarize the billing information for the given time range"
+    client: BillingSummaryClientService = build_client(ServiceType.BILLING_SUMMARY)
+    group_id = None
+    project_id = None
+    if not view_organization:
+        project_id = get_current_project_id()
+        if project_id is None:
+            secho_error_and_exit("'project_id' is not set!")
```
Comment (ktaebum) [inline pfcli/billing.py:69]: `Project is not set` might be more intuitive?

**Review by kooyunmo — APPROVED** [review]

**Pull request merged**